repos/system_upgrade/el7toel8/actors/rpmscanner/libraries/library.py

```diff
@@ -0,0 +1,12 @@
+
+def get_package_repository_data(self):
+    """ Return dictionary mapping package name with repository from which it was installed """
+    # import has to be inside the function to obey troubles with non-existing
```

Comment:
```suggestion
    # import has to be inside the function to evade troubles with non-existing
```

Reply:
I would say 'avoid' would be the best word here :D